qtconsole/rich_jupyter_widget.py

```diff
@@ -185,7 +183,15 @@ def _handle_display_data(self, msg):
 
     def _append_latex(self, latex, before_prompt=False, metadata=None):
         """ Append latex data to the widget."""
-        self._append_png(latex_to_png(latex, wrap=False), before_prompt, metadata)
+        supported_latex = latex.startswith('$') and latex.endswith('$')
```

Comment:
this is missing the other mathmode-triggering syntaxes: https://github.com/jupyter/qtconsole/issues/56#issuecomment-178167408

we should add a more sophisticated check extracted to a utility function, e.g. called `is_latex_mathmode`